vita3k/renderer/src/gl/screen_render.cpp

```diff
@@ -179,7 +182,9 @@ void ScreenRenderer::render(const SceFVector2 &viewport_pos, const SceFVector2 &
     }
 
     glBindTexture(GL_TEXTURE_2D, texture);
+    glPolygonMode(GL_FRONT_AND_BACK, GL_FILL);
     glDrawArrays(GL_TRIANGLE_FAN, 0, 4);
+    glPolygonMode(GL_FRONT_AND_BACK, *last_polygon_mode);
```

Comment:
```suggestion
    glPolygonMode(GL_FRONT, last_polygon_mode[0]);
    glPolygonMode(GL_BACK, last_polygon_mode[1]);
```

Reply:
It does not work for me. I got error GL_INVALID_ENUM.
I suggest 
`    if (last_polygon_mode[0] == last_polygon_mode[1] || last_polygon_mode[1] == GL_NONE) {
        glPolygonMode(GL_FRONT_AND_BACK, last_polygon_mode[0]);
    } else {
        glPolygonMode(GL_FRONT, last_polygon_mode[0]);
        glPolygonMode(GL_BACK, last_polygon_mode[1]);
    }`
not sure if `last_polygon_mode[1] == GL_NONE` is needed but it does not hurt either.